app/controllers/application_controller.rb

```diff
@@ -11,6 +11,14 @@ class ApplicationController < ActionController::API
 
   protected
 
+  def coded_token
+    token = Base64.encode64(request.headers['Authorization'])
```

Comment:
Useless assignment to variable - `token`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.